extras/kms/migrations/sqlite/04_keys.up.sql

```diff
@@ -0,0 +1,155 @@
+create table kms_root_key (
```

Comment:
I might be missing it, but I believe that on the postgres side there are checks on the length of the private ID (must be >= 10) but on the sqlite side there isn't. They should probably behave the same way.

Reply:
ty!